webpack.common.js

```diff
@@ -73,7 +73,7 @@ const serverConfig = merge(config, {
     'pages/DashboardPage': './src/pages/DashboardPage.js',
     'utils/Card': './src/utils/Card.js',
     'utils/draftutil': './src/utils/draftutil.js',
-    'utils/Filter': './src/utils/Filter.js',
+    'filtering/filterCards': './src/filtering/filterCards.js',
```

Comment:
Can you follow the capitalized naming convention for all of the files in `src` here?